Jenkinsfile.yml

```diff
@@ -11,7 +11,7 @@ node('ubuntu18.04-OnDemand'){
   stage('build'){
     sh 'mvn clean install'
     print "Archive the artifacts"
-    sh 'tar -czvf demo.tar.gz demo/'
+    sh 'tar -czvf demo.tar.gz component-samples/demo/'
```

Comment:
The generated demo.tar.gz now extracts to component-samples/demo .. 

Use `sh 'cd component-samples && tar -czvf demo.tar.gz demo && mv demo.tar.gz ..'`